src/components/WebexLocalMedia/WebexLocalMedia.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import jsonData from '../../data';
+import {WebexJSONAdapter} from '../../adapters';
+import {WebexLocalMedia, WebexDataProvider} from '../';
+
+// Setup for the stories
+const stories = storiesOf('Webex Local Media', module);
+const webexAdapter = new WebexJSONAdapter(jsonData);
+const wrapperStyle = {height: '500px', width: '800px', border: '1px solid black'};
+
+stories.add('local video', () => (
```

Comment:
We can call this "enabled" since the component name is already Local Media
```suggestion
stories.add('enabled', () => (
```